tools/legacy/Microsoft.WindowsAzure.Build.Tasks/Build.Tasks/SDKCategorizeProjects.cs

```diff
@@ -332,7 +332,7 @@ private bool IsTargetFxSupported(string fxMoniker, out TargetFrameworkMoniker ta
                     fxSupported = true;
                     break;
```

Comment:
It is probably not worth touching things under legacy. I actually think we should try to delete these before fixing them, I don't think they are used any longer.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/25494